File: pset6/mario.py
import cs50 

def main():
    Height = -1
    while True:
            print("Height: ", end = "")
            Height = cs50.get_int()
            if Height >= 0 and Height <= 23:
                break
    
    for i in range(Height):
        for j in range(Height - i - 1):
            print(' ', end = "")
        for j in range(i + 1):
            print("#", end = "")
        print('  ', end = "")
        for j in range(i + 1):
            print('#', end = "")
        print('')
        
main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void) 
{   
    int Height;
    do {printf("Height: ");
        Height = get_int();}
    while (Height > 23 || Height <= -1);
    for (int i=0; i < Height; i++){
        for (int s = 0; s < Height - i - 1; s++){
            printf(" ");
        }
        for (int x = 0; x < i + 1 ; x++){
            printf("#");
        }
        printf("  ");
        for (int x = 0; x < i + 1; x++){
            printf("#");
        }
        printf("\n");
    }
}